### ISSUE TITLE
Add documentation for new PlayServices/PlayWeb sbt plugins

### ISSUE THREAD

**Issue by marcospereira** [playframework/playframework#8039]
This is follow up of #8011. We need to add documentation highlighting these changes and also how to use them.

**Comment by gmethvin** [comment]
We should probably add some example projects around this as well.

**Comment by marcospereira** [comment]
> We should probably add some example projects around this as well.

Yes, I was considering to duplicate/adapt the REST samples to use PlayService. :-)

**Comment by ignasi35** [comment]
> Yes, I was considering to duplicate/adapt the REST samples to use PlayService. :-)

https://github.com/playframework/play-scala-rest-api-example/pull/144
